src/connector/src/sink/dynamodb.rs

```diff
@@ -185,6 +215,7 @@ struct DynamoDbPayloadWriter {
     client: Client,
```

Comment:
The `request_items` is always taken out and got clear when we write a chunk. We can change to remove the field from the struct and create a new vec and consume it in the `write_chunk`.